platformio.ini

```diff
@@ -20,4 +20,8 @@ lib_deps =
   ESPAsyncTCP
   ESP Async WebServer
 	MAX31855
+  adafruit/MAX6675 library @ ^1.1.0
```

Comment:
One last thing is left to do:

There needs to be a `[platformio]` section with `env_default` field.
Also please add a second env section named e.g. `env:reflow_{name of MAX chip}` and copy your changes there.
That way there will be two environments, one for MAX31855 and one for MAX6675 and they will be using the flags/dependencies that are needed only for that build environment.
That is helpful, because one wouldn't need to edit `platformio.ini` every time they want to build against different chips and also it would be very clear what settings are needed for what chip.

Also, since this project also builds with Wemoss D1 mini, I'll add an environment for that one too after this change lands.

One more great thing of having multiple build environments - automated builds that I'm planning to configure after I finish a blog article about this project :)

Reply:
done.